### ISSUE TITLE
importing estnltk fails: cannot import name 'Layer' from partially initialized module 'estnltk_core.layer.layer' 

### ISSUE THREAD

**Issue by milosh-dev** [estnltk/estnltk#117]
Fresh install via pip install estnltk==1.7.1

importing library Fails with the following error. 
```
from estnltk import Text
Traceback (most recent call last):
  File "<pyshell>", line 1, in <module>
  File "/home/raoul/.local/lib/python3.8/site-packages/estnltk/__init__.py", line 15, in <module>
    from estnltk_core.layer.layer import Layer
  File "/home/raoul/.local/lib/python3.8/site-packages/estnltk_core/layer/layer.py", line 10, in <module>
    from estnltk_core import BaseSpan, ElementaryBaseSpan, EnvelopingBaseSpan
  File "/home/raoul/.local/lib/python3.8/site-packages/estnltk_core/__init__.py", line 13, in <module>
    from estnltk_core.layer.layer import Layer
ImportError: cannot import name 'Layer' from partially initialized module 'estnltk_core.layer.layer' (most likely due to a circular import) (/home/raoul/.local/lib/python3.8/site-packages/estnltk_core/layer/layer.py)
```

**Comment by soras** [comment]
Thank you for the report!
Can you specify, on which platform / operating system this error occurrs?

**Comment by milosh-dev** [comment]
Tiba kummaline muidugi on, et kaks eestlast räägivad eesti keelt analüüsivast tarkvarast inglise keeles, aga olgu...

The OS is Ubuntu 20.04. I was able to solve the issue, but cannot fully reconstruct, all the steps that I took to remedy the issue. However the error boiled down to the issue described here: [https://stackoverflow.com/questions/74852225/attributeerror-module-numpy-has-no-attribute-typedict](url) and I had to downgrade numpy version to 1.21

Unfortunately cannot track down the library/line, which was referred to with the error.
